browser_patches/webkit/UPSTREAM_CONFIG.sh

```diff
@@ -1,3 +1,3 @@
 REMOTE_URL="https://github.com/webkit/webkit"
 BASE_BRANCH="master"
-BASE_REVISION="19d11039faa6e4da8cf7487e9c54a00a7c6e18e2"
+BASE_REVISION="822da5711ad337e1a7da82e813aa99e14fb79e49"
```

Comment:
Is this an intended change? Yury rolled webkit recently, so there should be no need.

Reply:
Is intended. `19d11039faa6e4da8cf7487e9c54a00a7c6e18e2` is webkit **r264022**, but for this patch we need at least webkit **r264092** (which its like 70 commits ahead). Here I used a commit a bit later than that: `822da5711ad337e1a7da82e813aa99e14fb79e49` (**r264108**) which was the last commit i could assert yesterday it was passing fine at the build.webkit.org CI